gymnasium/envs/mujoco/half_cheetah_v5.py

```diff
@@ -53,26 +53,28 @@ class HalfCheetahEnv(MujocoEnv, utils.EzPickle):
-    | --- | ------------------------------------ | ---- | --- | -------------------------------- | ----- | ------------------------ |
-    | 0   | z-coordinate of the front tip        | -Inf | Inf | rootz                            | slide | position (m)             |
-    | 1   | angle of the front tip               | -Inf | Inf | rooty                            | hinge | angle (rad)              |
-    | 2   | angle of the second rotor            | -Inf | Inf | bthigh                           | hinge | angle (rad)              |
-    | 3   | angle of the second rotor            | -Inf | Inf | bshin                            | hinge | angle (rad)              |
-    | 4   | velocity of the tip along the x-axis | -Inf | Inf | bfoot                            | hinge | angle (rad)              |
-    | 5   | velocity of the tip along the y-axis | -Inf | Inf | fthigh                           | hinge | angle (rad)              |
-    | 6   | angular velocity of front tip        | -Inf | Inf | fshin                            | hinge | angle (rad)              |
-    | 7   | angular velocity of second rotor     | -Inf | Inf | ffoot                            | hinge | angle (rad)              |
-    | 8   | x-coordinate of the front tip        | -Inf | Inf | rootx                            | slide | velocity (m/s)           |
-    | 9   | y-coordinate of the front tip        | -Inf | Inf | rootz                            | slide | velocity (m/s)           |
-    | 10  | angle of the front tip               | -Inf | Inf | rooty                            | hinge | angular velocity (rad/s) |
-    | 11  | angle of the second rotor            | -Inf | Inf | bthigh                           | hinge | angular velocity (rad/s) |
-    | 12  | angle of the second rotor            | -Inf | Inf | bshin                            | hinge | angular velocity (rad/s) |
-    | 13  | velocity of the tip along the x-axis | -Inf | Inf | bfoot                            | hinge | angular velocity (rad/s) |
-    | 14  | velocity of the tip along the y-axis | -Inf | Inf | fthigh                           | hinge | angular velocity (rad/s) |
-    | 15  | angular velocity of front tip        | -Inf | Inf | fshin                            | hinge | angular velocity (rad/s) |
-    | 16  | angular velocity of second rotor     | -Inf | Inf | ffoot                            | hinge | angular velocity (rad/s) |
-    | excluded |  x-coordinate of the front tip  | -Inf | Inf | rootx                            | slide | position (m)             |
+
+    | Num | Observation                                 | Min  | Max | Name (XML) | Joint | Type (Unit)                |
+    | --- | ------------------------------------------- | ---- | --- | ---------- | ----- | -------------------------- |
+    | excluded | x-coordinate of the front tip          | -Inf | Inf | rootx      | slide | position (m)               |
```

Comment:
Excluded observation go to the bottom of list

Reply:
Sorry but I don't understand what you mean by index vs mapping.
Also if you see the Hopper and walker, excluded is at first position.

Reply:
1) The Excluded observations are at the bottom, 
![image](https://github.com/Farama-Foundation/Gymnasium/assets/30759571/bcbd9074-e8ae-42b0-b6d3-bc9eca92fef1)

2) the index in `Num` is used to indicate the index of the observation, the observation with Num=0 should be in `observation[0]`, the observation with Num=1 should be in `observation[1]`, where `observation` is returned from `step` and `reset`

p.s. ignore the CI failing, so long as pre-commit passes it is fine, it fails for an unrelated to this PR problem